torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -395,7 +396,11 @@ const Tensor& resize_(
     SymIntArrayRef size,
```

Comment:
Couldn't the explicit copying just be done for this variable by making it take the SymIntArray by value instead of by reference? That would make an explicit copy and allow it take to ownership if it's an rvalue? Or is the issue that there is no implicit conversion to convert the reference to a value?